ui/k-info-panel/switch_info.kytos

```diff
@@ -173,6 +180,34 @@
        request.done(this.msg_state_success)
        request.fail(this.msg_state_failure)
      },
+     bt_request(){
+       var temp = {}
+       temp["interfaces"] = this.chosen_interface
+       let request = $.ajax({
+                        type: "POST",
+                        url: this.$kytos_server_api + "kytos/of_lldp/v1/interfaces/" + this.chosen_option.toLowerCase(),
+                        async: true,
+                        contentType: "application/json",
+                        data: JSON.stringify(temp),
+       });
+       var _this = this
+       request.done(function(data){
+         let notification = {
+           title: 'Interfaces ' + _this.chosen_option + 'd: Success',
+           description: 'The interfaces ' + temp["interfaces"] + ' is/are ' + _this.chosen_option.toLowerCase() + 'd to receive lldp packet',
```

Comment:
Great to have the information description, @Alopalao. One minor gotcha though the description in the docs was incorrect, so here it's supposed to be `to send` LLDP packets instead of `receive`, could you adapt it? Same thing in the request.fail description if could adapt there too.

Reply:
Updated on commit [a48b869d69b914523a9b9bd606c8cdc74059e42e](https://github.com/kytos-ng/topology/pull/81/commits/a48b869d69b914523a9b9bd606c8cdc74059e42e)